runtime/vm/OutOfLineINL_openj9_internal_foreign_abi_UpCallMHMetaData.cpp

```diff
@@ -65,16 +54,15 @@ OutOfLineINL_openj9_internal_foreign_abi_UpcallMHMetaData_resolveUpcallDataInfo(
 			J9VMCONSTANTPOOL_JDKINTERNALFOREIGNNATIVEMEMORYSEGMENTIMPL_LENGTH,
 			J9VMCONSTANTPOOL_JDKINTERNALFOREIGNNATIVEMEMORYSEGMENTIMPL_SCOPE
 			};
+	const int cpEntryNum = (int)(sizeof(cpIndex) / sizeof(cpIndex[0]));
 
 	VM_OutOfLineINL_Helpers::buildInternalNativeStackFrame(currentThread, method);
 	for (int i = 0; i < cpEntryNum; i++) {
```

Comment:
Please avoid the cast by using `size_t` instead of `int` (for `i` and `cpEntryNum`).

Reply:
Fixed as suggested above.